Pages/Hotels/Feedbackpage.js

```diff
@@ -17,44 +17,19 @@ class Feedbackpage{
     didYouAccomplishEnds = "']"
     emailId = "#email_address"
     thankyouMsg = "//h5[contains(text(),'THANK YOU')]"
-
-
-
-    privacyStLinks = "//a[contains(text(),'Privacy Statement')]"
-
-    async verifyTermsAndConditionsNewTab(){
-        const allHandles = await this.commands.getAllWindowHandles();
-        for(let handle of allHandles) {
-            await this.commands.switchToWindowHandle(handle);
-            const currentTitle = await this.commands.getWindowTitle();
-            if(currentTitle.includes('Terms of Service')){
-                return true;
-                break;
-            }
-        }
-    }
-    async clickPrivacyStatement(){
-        const links = await this.commands.findWebElements(this.privacyStLinks);
-        await this.commands.clickWebElement(links[0]);
-    }
-    
-    async verifyPrivacyStNewTab(){
-        const allHandles = await this.commands.getAllWindowHandles();
-        for(let handle of allHandles) {
-            await this.commands.switchToWindowHandle(handle);
-            const currentTitle = await this.commands.getWindowTitle();
-            if(currentTitle.includes('Deals & Discount')){
-                return true;
-                break;
-            }
-        }
-    }
-
-
     
     async verifyThankYouMsgDisplayed(){
         return await this.commands.isWebElementDisplayed(this.thankyouMsg);
-      
+        // const allHandles = await this.commands.getAllWindowHandles();
+        // for(let handle of allHandles) {
+        //     await this.commands.switchToWindowHandle(handle);
+        //     const currentTitle = await this.commands.getWindowTitle();
+        //     if(currentTitle.includes('DirectWord')){
+        //         await this.commands.switchToWindowHandle(handle);
+        //         await this.commands.clickWebElement(rating);
+        //         break;
+        //     }
+        // }
```

Comment:
There should not be any commented out code in repo. You can comment out code while running locally, but before push if the code is good un comment it or delete it.